### DISCUSSION TITLE
PagView如何在RecycleView中不暂停播放一次

### DISCUSSION THREAD

**Discussion by sansan-cc** [Tencent/libpag#1631] [General] [answered]
在RecycleView 中使用PagView，只播放一次并且功能依赖与播放结束监听，在列表末端不停的滑动，使PagView 在可见和不可见来回切换，那么它会不停地触发 resumeAnimator和pauseAnimator，并且是从头开始播放，那么在这种情景下，播放一次的PagView将一直不会触发结束监听。不知道这是个Bug还是设计机制就是这样。哪位大佬知道怎么解决这个问题。
版本 ： com.tencent.tav:libpag:4.1.49

**Comment by kevingpqi123** [comment, marked as answer]
如果你这边设置了只播放一次，如果没有播放完，来回滑动的过程中是会触发播放与暂停，直至一次播放完成，这是正常的逻辑
如果你不想在滑动的过程中频繁触发播放，直接在 Cell 不可见的时候 stop 就可以了，这样滑动的过程中就不会再触发播放逻辑

**Reply by sansan-cc** [reply]
有个疑问哈，来回滑动触发的暂停播放，播放是从头开始的，并不是从暂停的位置开始，是这样子设计的么。
其实我需求是动画不暂停的播放一次，即使是在隐藏之后。QAQ

**Reply by kevingpqi123** [reply]
不是从头开始的，是从上次暂停的位置开始的
